anta/tests/security.py

```diff
@@ -820,3 +820,41 @@ def test(self) -> None:
                     self.result.is_failure(
                         f"IPv4 security connection `source:{source_input} destination:{destination_input} vrf:{vrf}` for peer `{peer}` is not found."
                     )
+
+
+class VerifyHardwareEntropy(AntaTest):
+    """
+    Verifies hardware entropy generation is enabled on device.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if hardware entropy generation is enabled.
+    * Failure: The test will fail if hardware entropy generation is not enabled.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.security:
+      - VerifyHardwareEntropy:
+    ```
+    """
+
+    name = "VerifyHardwareEntropy"
+    description = "Verifies hardware entropy generation."
+    categories: ClassVar[list[str]] = ["security"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show management security")]
+
+    @AntaTest.anta_test
+    def test(self) -> None:
+        """Main test function for VerifyHardwareEntropy."""
+        self.result.is_success()
+
+        command_output = self.instance_commands[0].json_output
+
+        # Check if security parameters are configured
+        if not command_output:
+            self.result.is_failure("No cryptographic algorithm is configured.")
```

Comment:
Do we have this case that command reflect an empty output? If yes then its ok

Reply:
IMO, We don't have such case.
removing the check.
Thanks!